scalding-core/src/test/scala/com/twitter/scalding/source/typedtext/TypedTextTest.scala

```diff
@@ -0,0 +1,24 @@
+package com.twitter.scalding.source.typedtext
+
+import com.twitter.scalding._
+
+import org.scalatest.FunSuite
+import org.scalatest.junit.JUnitRunner
+
+case class Test1(a: Int, b: Long, c: Option[Double])
+case class Test2(one: Test1, d: String)
+
+class TypedTextTest extends FunSuite {
+  test("Test with a flat tuple") {
+    val source = TypedText.tsv[Test1]("myPath")
+    assert(source.sourceFields.size == 3)
+  }
+  test("Test with a nested tuple") {
+    val source = TypedText.tsv[Test2]("myPath")
```

Comment:
Would it be sufficient to add more tests exactly like these two, but with tuples and strings instead?

I was trying to write a test that reads in a TSV and checks that it read in the correct values. The only way I can think of doing that is by converting the `TypedPipe` to a list and checking for equality with the expected value.
However, it looks like I can't do that because the function to convert a pipe to an iterator/list is in `scalding-repl`.
Do you know of another way to test this?